### PR TITLE
 build with ModuleBuild installs dependencies 

### PR DESCRIPTION
**Pull request by JForst**
Hi, 

this is my fix for [this issue](https://github.com/tokuhirom/Minilla/issues/189)

Best regards

### CODE REVIEW

File: lib/Minilla/ModuleMaker/ModuleBuild.pm [hunk 1]
```diff
@@ -94,6 +94,36 @@ my %args = (
     configure_requires => {
         'Module::Build' => '0.4005',
     },
+
```
Comment (syohex) [inline lib/Minilla/ModuleMaker/ModuleBuild.pm]: Please remove trailing spaces.

**Comment by syohex** [comment]
Looks good to me. Could you squash commits ?

**Comment by JForst** [comment]
unfortunately i did all my changes in the master tree, should i squash and force this, or can you pull from a [branch](https://github.com/JForst/Minilla/tree/ModuleBuild_requirement)

**Comment by syohex** [comment]
I'll cherry-pick and squashed later.

**Pull request closed without merging**